src/main/java/uk/ac/cam/cl/dtg/segue/etl/ContentIndexer.java

```diff
@@ -107,18 +100,32 @@ void loadAndIndexContent(String version) throws Exception, VersionLockedExceptio
             Map<String, String> publishedUnits = new HashMap<>();
             Map<Content, List<String>> indexProblemCache = new HashMap<>();
 
+            long totalStartTime, startTime, endTime;
+
+            totalStartTime = System.nanoTime();
             buildGitContentIndex(version, true, contentCache, tagsList, allUnits, publishedUnits, indexProblemCache);
+            endTime = System.nanoTime();
+
+            log.info("Finished populating Git content cache, took: " + ((endTime - totalStartTime) / 1000000) + "ms");
```

Comment:
Felt the same way but erred on the side of "Oh it'll be fine its fairly clear what's going on" - I'll put this in since I'm implementing the extra bulk index by ID function 👍